compiler-rt/lib/msan/msan.cpp

```diff
@@ -352,10 +352,44 @@ void __sanitizer::BufferedStackTrace::UnwindImpl(
 
 using namespace __msan;
 
+#define PRINT_FAULTING_INSTRUCTION(instname)                            \
```

Comment:
`PRINT_FAULTING_INSTRUCTION_IF_REQUESTED`?

Why can't this be a real function? This won't show up in the stack trace?